<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealEventWindow</class>
 <widget class="QDialog" name="RealEventWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="btnRealEvent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReTurnMenu">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QToolButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RecogInfolabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
